cmd/registry/diff/differ.go

```diff
@@ -87,7 +87,7 @@ func addToDiffProto(diffProto *rpc.Diff, changePath *change) {
 	}
 }
 
-// getChanges creates a protodif report from a diff.Diff struct.
+// getChanges creates a protodiff report from a diff.Diff struct.
```

Comment:
I looked and couldn't find a `protodif` and guessed it was a typo for `protodiff`, but I could be wrong.